react/game/src/app/screens/About/index.js

```diff
@@ -1,9 +1,41 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import styles from './styles.scss';
 
 class About extends Component {
   render() {
-    return <h2>About</h2>;
+    const { user } = this.props;
+    return (
+      <Fragment>
+        <h2 className={styles.title}>About - {user.name}</h2>
+        <div className={styles.about}>
+          <p className={styles.paragraph}>
+            <span>Username:</span> {user.username}
+          </p>
+          <p className={styles.paragraph}>
+            <span>Date of birth:</span> {user.date_of_birth}
+          </p>
+          <p className={styles.paragraph}>
+            <span>e-Mail:</span> {user.email}
+          </p>
+          <p className={styles.paragraph}>
+            <span>Resume:</span> {user.about}
+          </p>
+        </div>
+      </Fragment>
+    );
   }
 }
 
-export default About;
+About.propTypes = {
+  user: PropTypes.shape(Object)
```

Comment:
U must define the correct propType. As `user` is a common value, and maybe u need to read in many screens, the correct way is extract this propType in a separate folder named `src/types` or `src/propTypes`, and then you shoudl use it in each component that uses this value of the store. Define a file per reducer. 
In this case, u need to extract the propType in `src/types/auth.js`, the import here and use it